src/current/v25.1/monitor-and-analyze-transaction-contention.md

```diff
@@ -0,0 +1,586 @@
+
+## Monitor using metrics
+
+As part of normal operation, CockroachDB continuously records metrics that are often useful in troubleshooting performance. The way metrics are exposed depends on the deployment type. Refer to:
+
+- [Export Metrics From a CockroachDB Standard Cluster]({% link cockroachcloud/export-metrics.md %})
+- [Export Metrics From a CockroachDB Advanced Cluster]({% link cockroachcloud/export-metrics-advanced.md %})
+- [Prometheus endpoint for a self-hosted cluster]({% link {{ page.version.version }}/monitoring-and-alerting.md %}#prometheus-endpoint)
+
+The following metrics related to contention are available:
+
+CockroachDB Metric Name | Description | Type | Unit
+------------------------|-------------|------|------
+`sql.distsql.contended_queries.count` | Number of SQL queries that experienced contention<br><br>Note: Only lock contention counts are included in this metric. | COUNTER | COUNT
+`sql.distsql.cumulative_contention_nanos` | Cumulative contention across all queries (in nanoseconds)<br><br>Note: Only lock contention durations are included in this metric. | COUNTER | NANOSECOND 
+`txn.restarts` | Number of restarted KV transactions | HISTOGRAM | COUNT
+`txn.restarts.asyncwritefailure` | Number of restarts due to async consensus writes that failed to leave intents | COUNTER | COUNT
+`txn.restarts.commitdeadlineexceeded` | Number of restarts due to a transaction exceeding its deadline | COUNTER | COUNT
+`txn.restarts.readwithinuncertainty` | Number of restarts due to reading a new value within the uncertainty interval | COUNTER | COUNT
+`txn.restarts.serializable` | Number of restarts due to a forwarded commit timestamp and isolation=SERIALIZABLE | COUNTER | COUNT
+`txn.restarts.txnaborted` | Number of restarts due to an abort by a concurrent transaction (usually due to deadlock) | COUNTER | COUNT
+`txn.restarts.txnpush` | Number of restarts due to a transaction push failure | COUNTER | COUNT
+`txn.restarts.unknown` | Number of restarts due to a unknown reasons | COUNTER | COUNT
+`txn.restarts.writetooold` | Number of restarts due to a concurrent writer committing first | COUNTER | COUNT
+
+{{site.data.alerts.callout_info}}
+Since these metrics are not categorized by database, interpreting them can be challenging.
```

Comment:
Do we really need to call out it's challenging? Users can correlate these metrics with insights and SQL activity and find the 'database'